bidict/_bidict.py

```diff
@@ -73,10 +79,10 @@ def clear(self):
         self._inv.clear()
 
     def pop(self, key, *args):
-        """Like :py:meth:`dict.pop`, keeping bidirectionality intact."""
-        ln = len(args) + 1
-        if ln > 2:
-            raise TypeError('pop expected at most 2 arguments, got %d' % ln)
+        """Like :py:meth:`dict.pop`, maintaining bidirectionality."""
+        l = len(args) + 1
```

Comment:
Thanks for flagging. I was conscious of that when I wrote it but made the decision to do it anyway. What would you name this variable?